### PR TITLE
Add a small text under the table. #102

### LINKED ISSUE #102: Add a small text under the table.
Under the table add a small text in grey to link to the hidden page that is the discount.

Screen Shot 2020-12-05 at 12 13 47 PM

### PR DESCRIPTION
**Pull request by edhenderson**
- closes #102

### CODE REVIEW

File: _src/_includes/pricing.html [hunk 1]
```diff
@@ -51,6 +51,11 @@ <h2>
             </tbody>
           </table>
         </div>
+        <p class="text-center">
+          <small class="text-muted text-uppercase">
```
Comment (davidgatti) [inline _src/_includes/pricing.html:55]:
```suggestion
          <small class="text-muted">
```

No upper case.

**Review by davidgatti — CHANGES_REQUESTED** [review]
One small change

**Comment by edhenderson** [comment]
@davidgatti OK, I had just gone off your diagram, it was uppercase.

**Comment by edhenderson** [comment]
🏓  Good for another look.

**Pull request merged**